oozie-to-airflow/mappers/subworkflow_mapper.py

```diff
@@ -68,9 +69,10 @@ def __init__(
     def _parse_oozie_node(self):
         app_path = self.oozie_node.find("app-path").text
         app_path = el_utils.replace_el_with_var(app_path, params=self.params, quote=False)
+        self.app_name = app_path.rsplit("/", 1)[1]
```

Comment:
```suggestion
        _, _, self.app_name = app_path.rpartition("/")
```
This is more like Python, but the actual solution is also good.

Reply:
Thanks, I changed it.